.github/workflows/test_comprehensive.yml

```diff
@@ -133,7 +133,7 @@ jobs:
 
       - name: Install tox-conda
         if: runner.os == 'macOS' && matrix.python == '3.9'
-        run: pip install tox-conda>=0.5.0
+        run: pip install git+https://github.com/tlambert-forks/tox-conda.git
```

Comment:
Can you add a quick comment here why we're using your fork, could just reference this PR/ needed PR so we remember in case this takes a long time to get resolved